specification/e-referrals-service-api.yaml

```diff
@@ -167,40 +167,33 @@ paths:
       security:
         - bearerAuth: []
       description: |
-        ## Use case
-        As a Referring Clinician or Referring Clinician Administrator,
-        Having identified suitable Services for my patient, I want to create a referral for my patient and associate the Referral with a list of appropriate services
-        so I can initiate my Patient's referral pathway; allowing the Patient to book from one of the clinically appropriate services shortlisted.
-        
-        ## You can use this endpoint to
-        - Create a new Referral in e-RS for a single Patient if you are a Referring Clinician or Referring Clinician Admin user.
-        - Associate the Referral with a list of Services that are suitable to the patient's current care needs. This is called a "Shortlist" of Services.
-        - Set a "priority" for the Referral indicating how quickly the Referral needs to be dealt with.
-        - Detail the criteria used to identify suitable Services.
-        - Indicate who the Referring Clinician is.
-          - If you are authenticated as Referring Clinician Admin when using this endpoint, you must indicate who you the Referral is being created on behalf of. This is known as the "Referring Clinician".
-        - Provide an indication as to whether or not you will add a Referrer Letter to the Referral for a Service to review.
-          - This may be dependent on which services are are on your Shortlist.
-          - The action of associating a Referral Letter to a Referral is completed via another endpoint.
-        - Indicate if you want a reminder letter to be sent to the Patient if they don't book/defer appointment within "x" days. The reminder letter sent prompts the Patient to book their appointment.
-          - "0" = no reminder letter.
-          - A value of 1 to 180 can be used to set the number of days between creating the Referral and a letter being sent to the Patient.
-        
-        ## After successful creation of the Referral
-        - The Referral can be accessed in the e-RS Professional Application or, from another endpoint.
-        - Other endpoints can be used to progress or amend your Referral.
+        ## Overview
+        Use this endpoint to create a referral by choosing a shortlist of bookable services.
+
+        ## Pre-requisites
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
         
+        You need to have identified a `patient`, for example via [Personal Demographic Service (PDS)](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir).
```

Comment:
may be an off-the-wall suggestion, but since we're already preprocessing this YAML to stick it togther, couldn't we externalise base URLs like this and declare them in one place?

Reply:
I've done some simple placeholder replacement as discussed, it seem to work well and it certainly is going to save us some trouble in the long run.